js/templates/search.es6.js

```diff
@@ -1,14 +1,11 @@
 const bel = require('bel');
 
 module.exports = function () {
-    return bel`<div class="js-search js-menu-section">
-        <form class="js-search-form" name="x">
-          <input type="text" autocomplete="off" placeholder="Search DuckDuckGo" name="q" class="js-search-input" value="${this.model.searchText}" />
```

Comment:
i meant no effect in *our* codebase... do we want chrome autocompleting our search box? don't you think that's a bad experience? was imagining we would call out to DDG autocomplete service for suggestions. if chrome fills it in from browser history, dont you think some users will think we are tracking them? also, suggestions will be just from user history, not actual DDG autocomplete results. overall seems like bad ux, but will defer to you.